src/qibolab/native.py

```diff
@@ -348,8 +346,18 @@ class TwoQubitNatives:
     """Container with the native two-qubit gates acting on a specific pair of
     qubits."""
 
-    CZ: Optional[NativeSequence] = None
-    iSWAP: Optional[NativeSequence] = None
+    CZ: Optional[NativeSequence] = field(default=None, metadata={"symmetric": True})
+    CNOT: Optional[NativeSequence] = field(default=None, metadata={"symmetric": False})
+    iSWAP: Optional[NativeSequence] = field(default=None, metadata={"symmetric": True})
```

Comment:
I'm not completely sure that using the field `metadata` is advisable.

Would you explain your choice?

Reply:
I.e. I see your point of keeping the static information together with the field definition, that is fine.

However, the `TwoQubitNatives` is becoming something like a container of enum values (actually, enum can not contain further data in Python, so it's more something like a container of homogeneous objects, i.e. same parent class).

However, if this is working, even fine like this. We could rework the `TwoQubiteNatives` class later on.